Makefile

```diff
@@ -0,0 +1,33 @@
+# It's necessary to set this because some environments don't link sh -> bash.
+SHELL := /bin/bash
+
+include ./make/verbose.mk
+.DEFAULT_GOAL := help
+include ./make/help.mk
+include ./make/out.mk
```

Comment:
Nothing is generated inside `/out` .
Do we need this?

Reply:
It's used by `check-go-fmt`.